tools/syz-execprog/execprog.go

```diff
@@ -303,6 +303,12 @@ func createConfig(target *prog.Target, entries []*prog.LogEntry,
 		execOpts.FaultCall = *flagFaultCall
 		execOpts.FaultNth = *flagFaultNth
 	}
+	for _, entry := range entries {
+		if entry.Fault {
+			config.Flags |= ipc.FlagEnableFault
```

Comment:
What does this do? I see it sets flag_enable_fault_injection in executor, but I don't see it's used anywhere.